src/test/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjectionTest.java

```diff
@@ -0,0 +1,592 @@
+package org.sasanlabs.service.vulnerability.commandInjection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+import static org.sasanlabs.vulnerability.utils.Constants.LOCALHOST;
+
+import java.io.IOException;
+import java.net.URI;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.util.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.sasanlabs.service.exception.ServiceApplicationException;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+
+/*
+ * @author Feng-Yuan Yang coolesaltw@hotmail.com
+ */
+public class CommandInjectionTest {
+
+    private CommandInjection commandInjection;
+    private static final String ipAddress = "10.29.28.27";
+    private static final String mockedSuccessPingResponse = "PING SUCCESS";
+
+    @BeforeEach
+    void setUp() throws IOException {
+        commandInjection = Mockito.spy(new CommandInjection());
+
+        StringBuilder stringBuilder = new StringBuilder().append(mockedSuccessPingResponse);
+
+        // Real localhost ping is not able to operate in CI, Mock instead
+        doReturn(stringBuilder)
+                .when(commandInjection)
+                .getResponseFromPingCommand(eq(LOCALHOST), eq(true));
+
+        doReturn(stringBuilder)
```

Comment:
if done the above mentioned changes this is not required.

Reply:
This mock and return is required, otherwise level 6 will get into do SH command

Reply:
aah got it because level 6 checks for both localhost and some ip address and you have added check for both.

thanks,
Karan